site/content/docs/5.2/forms/form-control.md

```diff
@@ -52,6 +52,11 @@ Add the `readonly` boolean attribute on an input to prevent modification of the
 
 If you want to have `<input readonly>` elements in your form styled as plain text, replace `.form-control` with `.form-control-plaintext` to remove the default form field styling and preserve the correct `margin` and `padding`.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This input readonly plain text element can not be used with this design in your forms because it does no respect the Orange Design System specifications. If you really need it, before using ask the Boosted team on Mattermost to have it modified by one of its designers.
```

Comment:
Hmm, IMO we can't say that in our doc: "This input readonly plain text element can not be used with this design in your forms because it does no respect the Orange Design System specifications.".

We need to provide a solution. Meaning that the designers need to work on it if they are not satisfied with our current implementation.

`<input readonly>` is perfectly legal so they must update their design guidelines to explain how it should be rendered.

Reply:
TO DO: ask design for a design for email@example.com + modify the message along to say that the form can't be used with labels on the side and direct to DSM/Forms + maybe add an example in Example page in Boosted

Reply:
Any update on it @isabellechanclou ? I still think it might be removed once #1813 is resolved and so we might tackle #1813 before next release so we might drop it now!